solc/CommandLineInterface.cpp

```diff
@@ -534,6 +537,10 @@ Allowed options)",
 		)
 		(g_argGas.c_str(), "Print an estimate of the maximal gas usage for each function.")
 		(
+			g_argStandardJSON.c_str(),
+			"Switch to Standard JSON input / output mode, ignoring all options."
```

Comment:
In the future, I think we should also allow to read the json from a file and not just from stdin. Can you add an explanation that we expect the input to come from stdin and write the output to stdout?